src/main/java/com/ideal/studentlog/helpers/dtos/TestDTO.java

```diff
@@ -11,12 +12,13 @@
 public class TestDTO {
 
     @NonNull
-    String subject;
+    @Schema(description = "Subject", example = "Bengali")
```

Comment:
Please create the branch from the latest main to avoid conflicts

Reply:
Done